src/main/kotlin/com/coder/toolbox/store/CoderSettingsStore.kt

```diff
@@ -0,0 +1,213 @@
+package com.coder.toolbox.store
+
+import com.coder.toolbox.settings.CTLSSettings
+import com.coder.toolbox.settings.CoderSettings
+import com.coder.toolbox.settings.Environment
+import com.coder.toolbox.settings.SettingSource
+import com.coder.toolbox.util.Arch
+import com.coder.toolbox.util.OS
+import com.coder.toolbox.util.getArch
+import com.coder.toolbox.util.getOS
+import com.jetbrains.toolbox.api.core.PluginSettingsStore
+import com.jetbrains.toolbox.api.core.diagnostics.Logger
+import java.nio.file.Path
+import java.nio.file.Paths
+
+class CoderSettingsStore(
+    private val store: PluginSettingsStore,
+    private val env: Environment = Environment(),
+    private val logger: Logger
+) {
+    private var backingSettings = CoderSettings(
+        defaultURL = store[DEFAULT_URL],
+        binarySource = store[BINARY_SOURCE],
+        binaryDirectory = store[BINARY_DIRECTORY],
+        defaultCliBinaryNameByOsAndArch = getCoderCLIForOS(getOS(), getArch()),
+        binaryName = store[BINARY_NAME] ?: getCoderCLIForOS(getOS(), getArch()),
+        dataDirectory = store[DATA_DIRECTORY],
+        globalDataDirectory = getDefaultGlobalDataDir().normalize().toString(),
+        globalConfigDir = getDefaultGlobalConfigDir().normalize().toString(),
+        enableDownloads = store[ENABLE_DOWNLOADS]?.toBooleanStrictOrNull() ?: true,
+        enableBinaryDirectoryFallback = store[ENABLE_BINARY_DIR_FALLBACK]?.toBooleanStrictOrNull() ?: false,
+        headerCommand = store[HEADER_COMMAND],
+        tls = CTLSSettings(
+            certPath = store[TLS_CERT_PATH],
+            keyPath = store[TLS_KEY_PATH],
+            caPath = store[TLS_CA_PATH],
+            altHostname = store[TLS_ALTERNATE_HOSTNAME]
+        ),
+        disableAutostart = store[DISABLE_AUTOSTART]?.toBooleanStrictOrNull() ?: (getOS() == OS.MAC),
+        isSshWildcardConfigEnabled = store[ENABLE_SSH_WILDCARD_CONFIG]?.toBooleanStrictOrNull() ?: false,
```

Comment:
```suggestion
        isSshWildcardConfigEnabled = store[ENABLE_SSH_WILDCARD_CONFIG]?.toBooleanStrictOrNull() ?: true,
```
Let's set this to true by default